Josefina/Controllers/ProjectTicketsAPIController.cs

```diff
@@ -98,6 +98,18 @@ public OrdersViewModel GetOrdersViewModel(int projectId)
                                 orderViewModel.Ordered = ticketOrder.Created;
                                 orderViewModel.PaidDate = ticketOrder.PaidDate;
                                 orderViewModel.TicketOrderID = ticketOrder.TicketOrderID;
+                                orderViewModel.Names = "";
+
+                                var ticketCategoryOrders = context.TicketCategoryOrders.Where(tco => tco.TicketOrderID == ticketOrder.TicketOrderID);
+
+                                foreach (TicketCategoryOrder ticketCategoryOrder in ticketCategoryOrders) {
+                                    var ticketItems = context.TicketItems.Where(ti => ti.TicketCategoryOrderID == ticketCategoryOrder.TicketCategoryOrderID);
+
+                                    foreach (TicketItem ticketItem in ticketItems) {
+                                        orderViewModel.Names += (orderViewModel.Names.Length == 0) ? ticketItem.Name : (", " + ticketItem.Name);

```

Comment:
As a tempory solution it's ok. But from user experience I woud go in the future with pop-up window that opens when you click on the detail button. 
This solution will result in column that could contain quite long text that will overlap with column width.